notebooks/lumigator_demo.py

```diff
@@ -241,7 +241,7 @@ def parse_model_results(results):
 
             row[column] = value
 
-        row["RAM_GB"] = mi[mi.model_name == model_name]['RAM_GB'].values[0]
+        row["RAM_GB"] = mi[mi.model_name == model_name]['RAM_GB'].to_numpy()[0]
```

Comment:
we'll want to check this one just in case

Reply:
Thanks @veekaybee ! Yes, it makes sense and I tested it, I confirm it works as expected 👍